routes/profile.php

```diff
@@ -161,6 +161,13 @@
 
 /*
 --------------------------------------------------------------------------
-Google2FA
+PayPal API
 --------------------------------------------------------------------------
 */
+
+Route::post('/process-transaction', 'PayPalController@processTransaction')
```

Comment:
Please prefix all these urls with `paypal/` to make it easier to see reading the code where it's going :)